Ginger/GingerCoreCommon/Repository/ALMLib/ALMDefectProfile.cs

```diff
@@ -74,5 +80,13 @@ public override string ItemName
                 _ALMDefectProfile = value;
             }
         }
+
+        public override void PostSerialization()
+        {
+            string almType = RepositoryItemHelper.RepositoryItemFactory.GetALMConfig();
```

Comment:
@Arvind 

You need to do this only if ALM type is null .
With current code, if your default ALM is qTest and on defect profile you will configure Jira.

Every time you will load the XML it will update defect profile and set it to Jira.
Please do this change and do detailed testing for xml loads.

Reply:
almType is not null, how to check this?

Reply:
Done